client/tokenized-express-checkout/transformers/stripe-to-wc.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
Copied from `client/tokenized-payment-request/transformers/stripe-to-wc.js` with some minor adjustments